src/arrow-sdk.ts

```diff
@@ -261,65 +261,87 @@ export async function getStrikeGrid(
 }
 
 /**
- * Submit an option order to the API to compute the live price and submit a transaction to the blockchain.
+ * Submit multiple option orders to the API to compute the live price and submit a transaction to the blockchain.
+ *
+ * @param deliverOptionParams[] Object containing parameters necessary to create an option order on Arrow.
+ * @param version Version of Arrow contract suite with which to interact. Default is V4.
+ * @returns Data object from API response that includes transaction hash and per-option execution price of the option transaction.
+ */
+export async function submitLongOptionOrder(
+    deliverOptionParams: DeliverOptionParams[],
+    version = DEFAULT_VERSION
+) {
+    if (!isValidVersion(version)) throw UNSUPPORTED_VERSION_ERROR
+
+    // Submit multiple option orders through API
+    let params: any[] = [];
+    
+    deliverOptionParams.map(function(order) 
+    {
+        params.push(
+        {
+            'order_type': order.orderType,
+            'ticker': order.ticker,
+            'expiration': order.expiration,
+            'strike': order.formattedStrike,
+            'contract_type': order.contractType,
+            'quantity': order.quantity,
+            'threshold_price': order.bigNumberThresholdPrice.toString(),
+            'hashed_params': order.hashedValues,
+            'signature': order.signature
+        })
+    })
+
+    const orderSubmissionResponse = await axios.post(
+        urls.api[version] + '/submit-order',
+        {
+            'params': params!
+        }
+    )
+
+    // Return all data from response
+    return orderSubmissionResponse.data
+}
+
+/**
+ * Submit a short option order to the API to compute the live price and submit a transaction to the blockchain.
  *
  * @param deliverOptionParams Object containing parameters necessary to create an option order on Arrow.
  * @param version Version of Arrow contract suite with which to interact. Default is V4.
  * @returns Data object from API response that includes transaction hash and per-option execution price of the option transaction.
  */
-export async function submitOptionOrder(
+export async function submitShortOptionOrder(
```

Comment:
Can we not support multiple short options at the same time?

Reply:
Smart contracts don't support it. Unless we call the current function multiple times. I did speak with Kai and he said he will work on adding that functionality to the smart contract for following iterations.